tests/backup/backup_delete_test.go

```diff
@@ -347,6 +347,7 @@ var _ = Describe("{DeleteIncrementalBackupsAndRecreateNew}", func() {
 				backupUID, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
 				log.FailOnError(err, "Failed while trying to get backup UID for - %s", backupName)
 				_, err = DeleteBackup(backupName, backupUID, orgID, ctx)
```

Comment:
What about this `err`? Don't we need to do `log.FailOnError`?

Reply:
Yes, how do you know if it failed to get backup driver or failed to do the op?